### DISCUSSION TITLE
Include short field name in keyword search in channel settings

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3822] [Feature Requests]
### Suggest an idea for ExpressionEngine

Here:
![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/a9d41bb0-a5e9-420a-8770-a9f55568dca1)

I have several fields called `Images`, but their short names are different: `company_images`, `news_images`, and so on. But I can't find anything by typing "news" in the search box, although short names are present. Can we please add them to the search?

**Comment by Yulyaswan** [comment]
PR: #3855
